src/chain/tezos/contracts/WrappedTezosHelper.ts

```diff
@@ -130,4 +161,171 @@ export namespace WrappedTezosHelper {
+     * @param signer A Signer for the sourceAddress.
+     * @param keystore A Keystore for the sourceAddress.
+     * @param ovenAddress The address of the oven contract. 
+     * @param fee The fee to use.
+     * @param amountMutez The amount of XTZ to withdraw, specified in mutez.
+     * @param gasLimit The gas limit to use.
+     * @param storageLimit The storage limit to use. 
+     * @returns A string representing the transaction hash.
+     */
+    export async function withdrawFromOven(
+        nodeUrl: string,
+        signer: Signer,
+        keystore: KeyStore,
+        ovenAddress: string,
+        fee: number,
+        amountMutez: number,
+        gasLimit: number,
+        storageLimit: number
+    ): Promise<string> {
+        const parameters = `${amountMutez}`
+
+        const nodeResult = await TezosNodeWriter.sendContractInvocationOperation(
+            nodeUrl,
+            signer,
+            keystore,
+            ovenAddress,
+            0,
+            fee,
+            storageLimit,
+            gasLimit,
+            'withdraw',
+            parameters,
+            TezosTypes.TezosParameterFormat.Michelson
+        )
+
+        return TezosContractUtils.clearRPCOperationGroupHash(nodeResult.operationGroupID);
+    }
+
+    export async function openOven(
+        nodeUrl: string,
+        signer: Signer,
+        keystore: KeyStore,
+        fee: number,
+        coreAddress: string,
+        gasLimit: number,
+        storageLimit: number
+    ): Promise<OpenOvenResult> {
+        const entryPoint = 'runEntrypointLambda'
+        const lambdaName = 'createOven'
+        const bytes = TezosMessageUtils.writePackedData(`Pair None "${keystore.publicKeyHash}"`, 'pair (option key_hash) address', TezosParameterFormat.Michelson)
+        const parameters = `Pair "${lambdaName}" 0x${bytes}`
+
+        const nodeResult = await TezosNodeWriter.sendContractInvocationOperation(
+            nodeUrl,
+            signer,
+            keystore,
+            coreAddress,
+            0,
+            fee,
+            storageLimit,
+            gasLimit,
+            entryPoint,
+            parameters,
+            TezosTypes.TezosParameterFormat.Michelson
+        )
+
+        const operationHash = TezosContractUtils.clearRPCOperationGroupHash(nodeResult.operationGroupID);
+        const ovenAddress = calculateContractAddress(operationHash, 0)
+        return {
+            operationHash,
+            ovenAddress
+        }
+    }
+
+    /**
+     * Calculate the address of a contract that was originated.
+     * 
+     * TODO(anonymoussprocket): This funcition is probably useful elsewhere in ConseilJS. Consider refactoring.
+     *
+     * @param operationHash The operation group hash.
+     * @param index The index of the origination operation in the operation group.
+     */
+    function calculateContractAddress(operationHash: string, index: number): string {
```

Comment:
This can go into 
[TezosMessageUtil](https://github.com/Cryptonomic/ConseilJS/blob/release/505/src/chain/tezos/TezosMessageUtil.ts) or into a new file in that directory like `TezosPlatformUtils.ts`